hack/build/setup-pac-integration.sh

```diff
@@ -48,19 +52,21 @@ setup-pac-app() (
         payload="{ \"iss\": $PAC_GITHUB_APP_ID, \"iat\": ${now}, \"exp\": $((now+10)) }"
 
         webhook_secret=$(openssl rand -hex 20)
+        token=$(sign rs256 "$payload" "$(echo "$PAC_GITHUB_APP_PRIVATE_KEY" | base64 -d)")
+        # return  sprayproxy host in a loop until it is available
+        while ! kubectl get -n sprayproxy route/sprayproxy-route >/dev/null 2>&1 ; do
```

Comment:
Maybe run this a reasonable number of times (100?) instead of an infinite loop? Even for a local script it's better experience instead of leaving it to the user to decide how long to wait.

Reply:
fixed